abstract-factory/src/main/java/com/iluwatar/abstractfactory/Kingdom.java

```diff
@@ -42,7 +42,7 @@ public Army getArmy() {
   }
 
   public void setKing(King king) {
-    this.king = king;
+    this.king = King;
```

Comment:
This is violating the principles of writing variables and methods.

Find the relevant information in our [developer wiki](https://github.com/iluwatar/java-design-patterns/wiki)